src/site/content/en/blog/conversion-measurement/index.md

```diff
@@ -0,0 +1,696 @@
+
+<div class="w-table-wrapper">
+  <table class="w-table--top-align">
+    <thead>
+      <tr>
+        <th><code>impressionexpiry</code></th>
+        <th>Depending on conversion time, a conversion report is sent (if the browser is open)...</th>
+        <th>Number of reporting windows</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>30 days, the default and maximum value</td>
+        <td>
+          <ul>
+            <li>2 days after the ad was clicked</li>
+            <li>or 7 days after ad click</li>
+            <li>or <code>impressionexpiry</code> = 30 days after ad click.</li>
+          </ul>
+        </td>
+        <td>3</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is between 7 and 30 days</td>
+        <td>
+          <ul>
+            <li>2 days after ad click</li>
+            <li>or 7 days after ad click</li>
+            <li>or <code>impressionexpiry</code> after ad click.</li>
+          </ul>
+        </td>
+        <td>3</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is between 2 and 7 days</td>
+        <td>
+          <ul>
+            <li>2 days after ad click</li>
+            <li>or <code>impressionexpiry</code> after ad click.</li>
+          </ul>
+        </td>
+        <td>2</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is under 2 days</td>
+        <td>
+            2 days after ad click
+        </td>
+        <td>1</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<figure class="w-figure">
+  <img src="./report-timing.jpg" alt="Chronology of what reports are sent when">
+</figure>
+
+See [Sending Scheduled
+Reports](https://github.com/WICG/conversion-measurement-api#sending-scheduled-reports) for more
+details on timing.
+
+## Example
+
+{% Banner 'info', 'body' %} To see this in action, try out the
+[demo](https://goo.gle/demo-event-level-conversion-measurement-api) ⚡️ and see the corresponding
+[code](https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement). {%
+endBanner %}
+
+Here's how the API records and reports a conversion. Note that this is how a click-to-convert flow
+would work with the current API. Future iterations of this API [may be different](#use-cases).
+
+### Ad click (steps 1 to 5)
+
+<figure class="w-figure">
+  <img src="./steps-1.jpg" alt="Diagram: ad click and click storage">
+</figure>
+
+An `<a>` ad element is loaded on a publisher site by `adtech.example` within an iframe.
+
+The adtech platform developers have configured the `<a>` element with conversion measurement
+attributes:
+
+```html
+<a
+  id="ad"
+  impressiondata="776f09351f5809c5"
+  conversiondestination="https://advertiser.example"
+  reportingorigin="https://adtech.example"
+  impressionexpiry="864000000"
+  href="https://advertiser.example/shoes07"
+  target="_blank"
+>
+  <img src="/images/shoe.jpg" alt="shoe" />
+</a>
+```
+
+This code specifies the following:
+
+{% Aside %}
```

Comment:
```suggestion
This code specifies the following:

<div class="w-table-wrapper">
  <table class="w-table--top-align">
    <thead>
      <tr>
        <th>Attribute</th>
        <th>Default value, maximum, minimum</th>
        <th>Example</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td><code>impressiondata</code> (required): a <b>64-bit</b> identifier to attach to an ad click.</td>
        <td>(no default)</td>
        <td>A dynamically generated click ID  such as a hex-encoded 64-bit integer:
`776f09351f5809c5`</td>
      </tr>
      <tr>
        <td><code>conversiondestination</code> (required): the <b>origin</b> where a conversion is expected for this ad.</td>
        <td>(no default)</td>
        <td><code>https://advertiser.example</code></td>
      </tr>
      <tr>
        <td><code>impressionexpiry</code> (optional): in milliseconds, the cutoff time for when conversions can be attributed to this ad.</td>
        <td>
          <p><code>2592000000</code> = 30 days (in milliseconds)</p>
          <p>Maximum: 30 days (in milliseconds)</p>
          <p>Minimum: 2 days (in milliseconds)</p>
        </td>
        <td>Ten days after click: <code>864000000</code></td>
      </tr>
      <tr>
        <td><code>reportingorigin</code> (optional): the destination for reporting confirmed conversions.</td>
        <td>Top-level origin of the page where the link element is added.</td>
        <td><code>https://adtech.example</code></td>
      </tr>
      <tr>
        <td><code>href</code>: the intended destination of the ad click.</td>
        <td><code>/</code></td>
        <td><code>https://advertiser.example/shoes07</code></td>
      </tr>
    </tbody>
  </table>
</div>

{% Aside %}
Some notes about the example:

- Note that `_target="parent"` is specified here, because in this example the ad is within an iframe.
```